cmd/cmd.go

```diff
@@ -587,6 +620,8 @@ func generateInteractive(cmd *cobra.Command, model string, wordWrap bool, format
 		fmt.Fprintln(os.Stderr, "  /set nowordwrap   Disable wordwrap")
 		fmt.Fprintln(os.Stderr, "  /set format json  Enable JSON mode")
 		fmt.Fprintln(os.Stderr, "  /set noformat     Disable formatting")
+		fmt.Fprintln(os.Stderr, "  /set image add <id> <file path>    Add image to chat")
+		fmt.Fprintln(os.Stderr, "  /set image rm <id>                 Add image to chat")
```

Comment:
Should this say "Remove image from chat"?

Reply:
These were mistakenly carried over from a merge.